src/views/home/post/attachments/index.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div>
+    <v-row align="center" justify="center">
+      <v-col
+        v-for="(attach, index) in getAttachments"
+        :key="index"
+        class="d-flex child-flex justify-center"
+        :md="
+          getAttachments.length < 2 ? 12 : getAttachments.length === 2 ? 6 : 4
+        "
+        sm="12"
+      >
+        <v-card flat tile>
+          <component :is="getMediaComponent(attach.type)" :url="attach.url" />
```

Comment:
We should link other files, if they're not an image and check of they have  the `preview` property set. I think you now only render images, right?

Reply:
That's right. I'll watch that with the preview